zerver/webhooks/front/doc.md

```diff
@@ -2,36 +2,26 @@ Front lets you manage all of your communication channels in one place,
 and helps your team collaborate around every message. Follow these steps
 to recieve Front notifications without leaving Zulip!
 
-{!create-stream.md!}
+1. {!create-stream.md!}
 
-{!create-bot-construct-url.md!}
+2. {!create-bot-construct-url.md!}
 
-Then, log into your Front account, and do the following:
+3. Log into your Front account, and open the **Settings**.
 
-1. Go to the **Settings** of your organization.
-   ![](/static/images/integrations/front/001.png)
+4. On the **Integrations** tab, find and enable the **Webhooks**
+integration.
 
-2. Enable webhook notifications.
-   ![](/static/images/integrations/front/002.png)
-   ![](/static/images/integrations/front/003.png)
+5. Then, open the **Rules** tab, and add a new rule: pick the types of
+events for which you want to receive notifications, and set the URL you
+construsted for the Front bot as a target webhook.
 
-3. Pick the types of events for which you want to receive notifications,
-and set the URL you construsted for the Front bot as a target webhook.
-   ![](/static/images/integrations/front/004.png)
-   ![](/static/images/integrations/front/005.png)
+6. Finally, open the **Settings** of your Zulip organization, and add a
+new filter on the **Filter settings** page. This step is necessary to set
+a link to a Front conversation as a name of the corresponding topic in
+Zulip.
+* Pattern: `cnv_(?P<id>[0-9a-z]+)`.
+* URL format string: `https://app.frontapp.com/open/cnv_%(id)s`.
```

Comment:
Okay, this one is a bit confusing, a couple of minor points:
* Don't use transition words like "Finally".
* I would replace the last sentence to something easier to read, its too confusing. Maybe `This step is necessary to map Front conversations to topics in Zulip`?